### ISSUE TITLE
DDOS OR SERVER LAG

### ISSUE THREAD

**Issue by KRALPATATES** [ValveSoftware/Dota2-Gameplay#6191]
### Description

DDOS or Server lag. Our entire team was lagging so bad and we lost cause of that. Some of the enemy had the same but 1 or 2 enemy was totally fine.

https://user-images.githubusercontent.com/121383664/209467118-246325fa-b8b1-4b52-9e39-aebf0469baba.mp4

i uploaded a clip. in this clip it wasnt that worse
watching the replay it looks everything is fine. but you will see people dc time to time for few secs

### Example Match ID (and possibly Timestamp)

6935788446 - 5:00-40:09

**Comment by shaileshgarg** [comment]
#6159

**Comment by KRALPATATES** [comment]
> #6159

Yeah the same thing. During team fights most of the time game was in slow motion and most of the animations were broken
